src/components/CheckboxInput/CheckboxInput.module.css

```diff
@@ -0,0 +1,90 @@
+/* TODO: uncomment once tokens migrated */
+
+/* @import '../../design-tokens/mixins.css'; */
+
+/* ------------------------------------*\
+    #CHECKBOX INPUT
+\*------------------------------------ */
+
+/**
+ * Wrapper for the checkbox, including the visually hidden input element.
+ * 1) Setting position is required to place the absolutely positioned input element in the same spot as the SVG 
+ *    when a (n^c)parent element in the tree is relatively positioned.
+ * 2) line-height is set to be same as checkbox size to equal checkbox size and prevent browser styling.
+ */
+.input__wrapper {
+  position: relative; /* 1 */
+  line-height: var(--checkbox-svg-size); /* 2 */
+}
+
+/**
+   * The visually hidden input element for the checkbox. The visual checkmark is provided by an svg element.
+   * 1) Removes default margins placed by browser for checkboxes.
+   * 2) Remove the checkbox from the page flow, positioning it on top of the SVG.
+   * 3) Set same dimensions as the CheckboxSvg element.
+   * 4) Hide the input element visually.
+   *    This ensures the checkbox is still "physically" present so that all users,
+   *    especially on touch screen readers, still interact with the real checkbox element
+   *    where it would naturally be present.
+   */
+.checkbox__input {
+  cursor: pointer;
+  margin: 0; /* 1 */
+  position: absolute; /* 2 */
+  height: var(--checkbox-svg-size); /* 3 */
+  width: var(--checkbox-svg-size);
+  opacity: 0; /* 4 */
+}
+
+/**
+   * The visible checkbox graphic.
+   * 1) Provides a focus indicator outline around the checkbox.
+   */
+.checkbox__input + svg {
+  vertical-align: top;
+  display: inline-block;
+
+  /* TODO: replace with token once migrated */
```

Comment:
I dunno if it's worth it leaving comments for all the tokens that need to be updated since there are so many. I was thinking that since all of the components are going to be using some token that isn't available yet, we can just copy over the old CSS files when the tokens are in or wait and replace the whole component when the big merge happens.

Reply:
Mmmm I think you're right. Should I remove these comments?